### DISCUSSION TITLE
Off Screen Drop down issues

### DISCUSSION THREAD

**Discussion by notpatern** [cefsharp/CefSharp#4907] [General]
When trying to open a Dropdown element on a webpage, it opens full screen and then disappears.
I tried to use the LifeSpanHandler to manage pop ups but Drop down elements are not Popups.

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

Please answer questions in the link
